Exiled.API/Features/Roles/Scp106Role.cs

```diff
@@ -28,7 +28,7 @@ public class Scp106Role : Role
         /// <summary>
         /// Gets the <see cref="Scp106PlayerScript"/> script for the role.
         /// </summary>
-        public Scp106PlayerScript Script => script ??= Owner.ReferenceHub.scp106PlayerScript;
+        public Scp106PlayerScript Script => script ? script : script = Owner.ReferenceHub.scp106PlayerScript;
```

Comment:
Why

Reply:
That check for unity objects, problem can be in caching objects that need to save not only in current scene(scp scripts dont need to save, but everything can be, if you sure that it won't lead to any errors - I revert it)

Reply:
I don't really get it.
`script ??= Owner.ReferenceHub.scp106PlayerScript` is the same as `script ? script : script = Owner.ReferenceHub.scp106PlayerScript`.

Reply:
No, check IL code

Reply:
Or ask Yamato, I already told him about diff

Reply:
Yeah whatever, it's the same thing in this case 🤷🏼‍♂️